test/unit/node/util.test.ts

```diff
@@ -482,3 +482,74 @@ describe("humanPath", () => {
     expect(actual).toBe(expected)
   })
 })
+
+describe("isWsl", () => {
+  describe("on Linux (microsoft)", () => {
+    const testDir = path.join(tmpdir, "tests", "isWsl-linux-microsoft")
+    let pathToFile = ""
+
+    beforeEach(async () => {
+      pathToFile = path.join(testDir, "/proc-version")
+      await fs.mkdir(testDir, { recursive: true })
+      await fs.writeFile(pathToFile, "microsoft")
+    })
+    afterEach(async () => {
+      await fs.rm(testDir, { recursive: true, force: true })
+    })
```

Comment:
I tried taking these lines and making a `setupIsWslTest` but Jest couldn't comprehend the inner `beforeEach` and `afterEach` so I ditched it.